docs/tools/tools_intro.md

```diff
@@ -19,12 +19,12 @@ The Mbed Online Compiler is our in-house IDE and should be familiar to anyone wh
 
 | Architecture version | Compiler |
```

Comment:
I would just get rid of the table. I do not see what useful information it gives and we are currently using the latest version of the toolchain.